### DISCUSSION TITLE
First slides are visible but Clicking Next arrow loads white screen even though HTML is there and loaded

### DISCUSSION THREAD

**Discussion by femina-clarion** [nolimits4web/swiper#4128] [Q&A]
Please help me!!!!!! I am really running out of my mind. 
here is my code 

[http://staging-new.shoresummerrentals.com/](http://staging-new.shoresummerrentals.com/ )

Check featured property .

var mySwiper = new Swiper('.swiper-container', {
                    // Optional parameters
                    direction: 'horizontal',
                    observer: true,
                    observeParents: true,
                    parallax:true,
                    speed: 1500,
                    simulateTouch : true,
                    loopFillGroupWithBlank: false,
                    grabCursor: true,
                    centeredSlides: true,
                    resistanceRatio:0,
                    // Enable lazy loading
                    // lazy: true,
                    pagination: {
                        el: '.swiper-pagination',
                        clickable: true,
                    },
                    watchSlidesProgress: true,
                    watchSlidesVisibility: true,
                    breakpoints: {
                        // when window width is >= 320px
                        320: {
                            slidesPerView: 1,
                            spaceBetween: 20
                        },
                        // when window width is >= 480px
                        480: {
                            slidesPerView: 1,
                            spaceBetween: 30
                        },
                        // when window width is >= 640px
                        640: {
                            slidesPerView: 1,
                            spaceBetween: 30,
                        },
                        768: {
                            slidesPerView: 4,
                            spaceBetween: 30,
                        },
                        1024: {
                            slidesPerView: 5,
                            spaceBetween: 30,
                            slidesPerGroup: 5
                        }
                    }
                });

**Comment by vltansky** [comment]
@femina-clarion still relevant? if yes, can you describe your problem?
